packages/manager/src/managers/telemetry/types.ts

```diff
@@ -360,6 +362,14 @@ type SliceMachinePostPushToastCtaClicked = SegmentEvent<
 	typeof SegmentEventType.postPush_toastCtaClicked
 >;
 
+type SliceMachineExperimentExposure = SegmentEvent<
+	typeof SegmentEventType.experiment_exposure,
+	{
+		flag_key: string;
+		variant?: string;
```

Comment:
💡 #suggestion

I think we can remove the `?` as variant should always be passed in our case, right?

Reply:
added it because amplitude `Variant` has it also with `?` and it made TypeScript not happy with line 310 in `TelemetryManager` (even though we check that `value` is thruthy so can't be `undefined` few lines up 🤷‍♂️ )

Reply:
yes, that worked, fixed